src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -49,6 +49,25 @@ static void Main()
                 );
         }
 
+        [Fact]
+        public void TestUnaryNegation()
```

Comment:
If this is testing something nullablewalker, it should have the `NonNullTypes` attribute.

Reply:
In DEBUG mode, we always run NullableWalker (regardless of NonNullTypes context). That's why that test was still useful to hit an assertion (now removed).

Reply:
@jcouv that means this is relying on a side effect to be a valid test, not actually doing anything useful directly.